src/common/TimeInterval.mjs

```diff
@@ -4,9 +4,9 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-const moment = require('moment')
```

Comment:
Should probably note that this commit doesn't fix anything server-side. NodeJS only allows importing ESM through dynamic imports. But, I didn't properly await the imports, so there were errors since the imports had a value of `Promise<Module>` rather than `Module`.

I fixed this in 9455185f929fdccab7f4e5d5ade83ee592fa3258 by converting things back to CommonJS. Luckily, I didn't get the previous errors I was getting with Vite because I updated Vite to a later version.